src/components/relative_time.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { formatDistanceToNow } from 'date-fns';
+
+export function RelativeTime({ datetime, addSuffix = true }: Object) {
+  return (
+    <time datetime={datetime.toISOString()} title={datetime.toString()}>
```

Comment:
I used the native JS date methods here, though there are probably equivalent functions in the `date-fns` module.

The `datetime` field contains structured data and should always be an ISO formatted date. The `title` can be any text; I chose the default `toString()` representation which in my locale looks like `Wed Jun 26 2024 14:28:48 GMT-0700 (Pacific Daylight Time)`. This method always displays the time in the browser's timezone. I don't think it changes the overall format based on the user's locale. Perhaps `toLocaleString()` would be better here? Open to feedback.